uktrade/data-hub-frontend
# Rebuild OMIS assignees form in React (remove vue.js)

## Description of change

As part of the work to remove the Vue.js Typeahead, we need to refactor the OMIS add assignees form.

## Test instructions

You should be able to add or amend assignees on an OMIS order.

Please note that there are several bugs in the existing page that were out of scope for this PR. They have been added to the maintenance backlog and will be picked up after Vue has been removed (see [this ticket](https://uktrade.atlassian.net/browse/CPS-321) and [this ticket](https://uktrade.atlassian.net/browse/CPS-322))

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/36161814/153196137-63bdaf50-996d-4fce-ac6c-724355e98772.png)

### After

<img width="1001" alt="Screenshot 2022-02-09 at 13 41 35" src="https://user-images.githubusercontent.com/36161814/153213008-0d91a416-26a5-4d36-bd5b-c8036af3c5cf.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
